examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -80,6 +85,12 @@ class _ButtonsDemoState extends State<ButtonsDemo> {
         widget: buildActionButton(),
         exampleCodeTag: _actionCode,
       ),
+      new ComponentDemoTabData(
+        tabName: 'iOS',
```

Comment:
For now, yes.  We'll probably change it later as we rationalize things more, but Cupertino is a good place to start.